api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -76,6 +86,8 @@ def open_soma(
     Args:
         census_version:
             The version of the Census, e.g. "latest" or "stable". Defaults to "stable".
+        mirror:
+            The mirror used to retrieve the Census. Defaults to the `us-west-2` mirror.
         uri:
             The URI containing the Census SOMA objects. If specified, will take precedence
             over ``census_version`` parameter.
```

Comment:
on the next arg, `context`, please add:  "... which will be used to open the SOMA object.  Optional, defaults to None."